make/scripts/compare-logger.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright (c) 2012, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2012, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This file was only renamed. Not sure if that counts. I understand that this is auto generated, so I'm fine with this if you are.